views/404.hbs

```diff
@@ -0,0 +1,11 @@
+{{!< default }}
+<div id="banner-info" class="section-container">
+    <div id="banner-sections-wrapper">
+        <div id="banner-left">
+            <div class="banner-content">
+              <h1>404: Page not found.</h1>
```

Comment:
Good enough(tm) for shield, but we may want to enlist Michelle or somebody for some better long-term copy, and possibly a link back to homepage or embedded search/subscribe capabilities so this feels less dead-endy.

Reply:
That's why I thought we could redirect to index for 404s :P 
I'm not a fan of the idea of putting "real" UI in a 404 page because of people possibly copying the link to share it.
Let me tinker a bit more with this before we merge.